packages/block-library/src/navigation/style.scss

```diff
@@ -61,9 +61,8 @@
 		& > li > ul {
 			margin: 0;
 			position: absolute;
-			background: #fff;
 			left: 0;
-			top: 100%;
+			top: 80%;
```

Comment:
What does this do?

Reply:
This moves sub-nav closer to the navigation.